src/utils/formatters.js

```diff
@@ -287,3 +287,20 @@ export const formatLocationFromSighting = (
 export const sanitizeTwitterHandle = value => {
   return value.replace(/^@/, '');
 };
+
+export const formatDuplicateLabel = (currentLabel, key, intl) => {
```

Comment:
this might never be used outside of bulk import. so instead of putting it in the common utils it might be better placed in the bulk import folder